rest/route/task.go

```diff
@@ -298,7 +298,10 @@ func (tep *taskExecutionPatchHandler) Run(ctx context.Context) gimlet.Responder
 				})
 			}
 		}
-		if err := data.SetTaskPriority(tep.task, tep.user.Username(), priority); err != nil {
+		if tep.task == nil {
```

Comment:
Did you encounter this? I would've thought it gets hit at line 268

Reply:
I was just carrying over the nil check that was previously in the SetTaskPriority method, but yes it looks like in this case we already do a nil check so I can remove this.